README.md

```diff
@@ -142,6 +142,21 @@ static:
     management_network_address: 172.23.99.0/24
 ```
 
+Example with openstack provider:  
+You will need to install vagrant openstack provider. For more information click [here](https://github.com/ggiamarchi/vagrant-openstack-provider).  
+Do not forget to set openstack API credentials.
+To use openstack provider as default look [here](https://www.vagrantup.com/docs/providers/default.html).  
+Tip: You can disable hostmanager by using settings.yaml
```

Comment:
Note that hostmanager is an optional plugin. By default it's not even present. I'd move this line to the section that tells uses it can be installed (https://github.com/theforeman/forklift/blob/04e95ed57dfba7bb2863efa9b064bcfed568d3b0/README.md#poor-mans-dns-aka-etchosts)